lib/Site.js

```diff
@@ -295,7 +296,7 @@ Site.prototype.collectUserDefinedVariablesMap = function () {
     const $ = cheerio.load(content);
     $.root().children().each(function () {
       const id = $(this).attr('id');
-      const html = $(this).html();
+      const html = nunjucks.renderString($(this).html(), userDefinedVariables);
```

Comment:
Shall we have a comment before this line? It won't be obvious why this was done.